error, ERR_MODULE_NOT_FOUND

macOs: 26.1
npm: 10.9.4
node: 22.21.1

eslint.config.mts

    import { defineConfig } from 'eslint/config';
    import eslint from '@eslint/js';
    import globals from 'globals';
    import tseslintParser from '@typescript-eslint/parser';

    export default defineConfig({
        extends: [
            eslint.configs.recommended,
        ],
        files: [
            '**/*.tsx',
        ],
        languageOptions: {
            ecmaVersion: 'latest',
            globals: globals.browser,
            parser: tseslintParser,
            parserOptions: {
                parser: tseslintParser,
                projectService: true,
                tsconfigRootDir: import.meta.dirname,
            },
            sourceType: 'module',
        },
        settings: {
            react: {
                version: 'detect',
            },
        },
    });


Error:

    You can also run this command directly using 'npx @eslint/config-inspector@latest' in the same directory as your configuration file.
    node:internal/modules/esm/resolve:274
        throw new ERR_MODULE_NOT_FOUND(
              ^

    Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/username/.npm/_npx/fadb285f08a830ab/node_modules/@eslint/config-inspector/dist/cli.js' imported from /Users/username/.npm/_npx/fadb285f08a830ab/node_modules/@eslint/config-inspector/bin.mjs
        at finalizeResolution (node:internal/modules/esm/resolve:274:11)
        at moduleResolve (node:internal/modules/esm/resolve:859:10)
        at defaultResolve (node:internal/modules/esm/resolve:983:11)
        at #cachedDefaultResolve (node:internal/modules/esm/loader:731:20)
        at ModuleLoader.resolve (node:internal/modules/esm/loader:708:38)
        at ModuleLoader.getModuleJobForImport (node:internal/modules/esm/loader:310:38)
        at onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:664:36)
        at TracingChannel.tracePromise (node:diagnostics_channel:350:14)
        at ModuleLoader.import (node:internal/modules/esm/loader:663:21)
        at defaultImportModuleDynamicallyForModule (node:internal/modules/esm/utils:222:31) {
      code: 'ERR_MODULE_NOT_FOUND',
      url: 'file:///Users/username/.npm/_npx/fadb285f08a830ab/node_modules/@eslint/config-inspector/dist/cli.js'
    }

    Node.js v22.21.1

--- a/bin.mjs
+++ b/bin.mjs
@@ -1,4 +1,4 @@
 #!/usr/bin/env node
 
 // eslint-disable-next-line antfu/no-top-level-await
-await import('./dist/cli.js')
+await import('./dist/cli.mjs')

